features/options/contract/version.feature

```diff
@@ -6,13 +6,6 @@ Feature: osrm-contract command line options: version
 
     Background:
         Given the profile "testbot"
-    
-    Scenario: osrm-contract - Version, short
```

Comment:
Ok. Follow-Up. Why aren't we choosing a different character than `v` instead? I don't feel removing the option is the way to go. I am thinking of heaps of scripts that contain short parameters in their calls. We would break all of them for this change.